README

```diff
@@ -202,10 +202,10 @@ Features
       Motherboard sensors: temperature, fan speed and voltage information,
       using mbmon(1).
 
-    - mcelogmon
-      Monitor machine check errors (hardware error detected by hardware
-      and reported to software) reported by mcelog and generate appropriate
-      notifications when machine check errors are detected.
+    - mcelog
+      Logs and accounts machine check exceptions (in particular memory, IO, and
```

Comment:
It logs only the  amount of corrected/uncorrected  memory related errors  per  memory SOCKET and CHANNEL.  No MCE error codes are logged if that's what you mean. Data source is equivalent to output of  'mcelog --client', for example:
```sh
[root@silpixa00392165 kmatczax]# mcelog --client
Memory errors
SOCKET 0 CHANNEL any DIMM any
corrected memory errors:
        0 total
        0 in 24h
uncorrected memory errors:
        8 total
        8 in 24h

SOCKET 0 CHANNEL 0 DIMM any
corrected memory errors:
        0 total
        0 in 24h
uncorrected memory errors:
        8 total
        8 in 24h
```